hooks/useExplorerConfiguration.ts

```diff
@@ -0,0 +1,32 @@
+import { useLocalStorage } from '@mantine/hooks';
+import { useMemo } from 'react';
+
+export enum Explorers {
+  SolanaFM = 'solanafm',
+  Solscan = 'solscan',
+  Xray = 'xray',
+  Solana = 'solana'
+}
+
+export function useExplorerConfiguration() {
+  const [explorer, setExplorer] = useLocalStorage<Explorers>({
+    key: 'meta-dao-explorer-configuration',
+    defaultValue: Explorers.SolanaFM,
+  });
+  const endpoint = useMemo(() => {
```

Comment:
I think most of them have some additional formatting needed (e.g. `explorer.solana.com/address` or `solscan.io/account`.
The explorer hook could also take as input the account or transaction hash and output the correct explorer link

Reply:
Will work to resolve this.